packages/components/src/popover/src/Popover.tsx

```diff
@@ -194,10 +198,26 @@ export function InnerPopover({
                     focusRingProps
                 )}
             >
-                {dismissButtonMarkup}
-                {headerSectionMarkup}
-                {content}
-                {footerSectionMarkup}
+                <Div className={cssModule(
+                    "o-ui-popover",
+                    dismissable && "dismissable"
+                )}
+                >
+                    {dismissButtonMarkup}
+                    {headerSectionMarkup}
+                    {content}
+                    {footerSectionMarkup}
+                </Div>
+
+                {arrowProps && <Div
```

Comment:
i don't really like that, introduce new prop HasArrow in addition to arrowProps?

Reply:
fixed